dnadoh/data_config.yml

```diff
@@ -0,0 +1,2 @@
+DNA: "ACGT"
+SNP_PROBS: !!python/tuple [0.50, 0.25, 0.13, 0.12]
```

Comment:
this is cool, is this some .yml specific syntax for defining types and defaults?

Reply:
When parsing yaml, I believe it automatically defaults to lists, so you need to specify that the objects are tuples. It's a little bit clunky, so we can try thinking of a more elegant solution!